queries/java/highlights.scm

```diff
@@ -51,40 +72,47 @@
   name: (identifier) @type)
 (enum_declaration
   name: (identifier) @type)
+(constructor_declaration
+  name: (identifier) @type)
+(type_identifier) @type
+
+
 
 ((field_access
   object: (identifier) @type)
- (#match? @type "^[A-Z]"))
+  (#match? @type "^[A-Z]"))
 ((scoped_identifier
   scope: (identifier) @type)
- (#match? @type "^[A-Z]"))
+  (#match? @type "^[A-Z]"))
 
-(constructor_declaration
-  name: (identifier) @type)
-
-(type_identifier) @type
-(boolean_type) @type.builtin
-(integral_type) @type.builtin
-(floating_point_type) @type.builtin
-(floating_point_type) @type.builtin
-(void_type) @type.builtin
+[
+(boolean_type)
+(integral_type)
+(floating_point_type)
+(floating_point_type)
```

Comment:
Floating_point_type is doubled in this query.

Reply:
done